environments/centos-8/image_tag

```diff
@@ -1,4 +1,4 @@
 # Bump this version whenever you've made a change and you want
 # to force users to re-pull the image (e.g. when your change adds
 # a feature that our scripts rely on, or is breaking).
-1
+2
```

Comment:
You need to rerun ./internal-scripts/generate-ci-cd-yaml.rb. For convenience you should setup the Git precommit hook locally: https://github.com/fullstaq-ruby/server-edition/blob/main/dev-handbook/dev-environment-setup.md#git-hooks

Reply:
Thanks! fixed at 86ce853